libs/financial-aid/shared/src/components/FileList/FileList.tsx

```diff
@@ -7,7 +7,7 @@ import {
   ApplicationFile,
   getFileSizeInKilo,
   getFileType,
-} from '@island.is/financial-aid/shared'
+} from '@island.is/financial-aid/shared/index'
```

Comment:
I can imagine you did not export this component in `index` to avoid circular imports. Splitting things into more libs can help with this.